crates/bevy_log/src/lib.rs

```diff
@@ -134,7 +134,68 @@ pub(crate) struct FlushGuard(SyncCell<tracing_chrome::FlushGuard>);
 ///         .run();
 /// }
 /// ```
+/// # Example Setup
 ///
+/// For a quick setup that enables all your loggings,while not showing all your dependencies log data
+/// you can configuere the plugin as below.
+///
+/// ```no_run
+/// fn main() {
+///     App::new()
+///         .add_plugins(DefaultPlugins.set(LogPlugin {
+///             filter: "warn,my_crate=trace".to_string(), //specific filters
+///             level: Level::TRACE,//Change this to be globally change levels
+///             ..Default::default()
+///         }))
+///         .add_systems(Startup, setup)
+///         .run();
+/// }
+/// ```
+/// The (Env) filter chooses whether to print the log, based on the most specific filters
```

Comment:
I don't understand what "Env" refers to here.

Reply:
Env refers specifically to this type of filter here https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html

Reply:
I think any filter beyond other then and Env filter is beyond the scope of this example.

I think I could reword to be something along the lines of "The filter (in this case an EnvFilter)"... then leave the rest of the sentence.

Reply:
That's much clearer to me :)

Reply:
Great I have changed the adjacent area.